app/src/main/AndroidManifest.xml

```diff
@@ -41,14 +41,8 @@
             android:label="@string/app_name"
             android:noHistory="false"
             android:screenOrientation="portrait"
-            android:theme="@style/AppTheme">
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
+            android:theme="@style/AppTheme" />
```

Comment:
do you wanna leave some custom intent action so we can test this when needed by launching it via adb ?

Reply:
Done